src/main/java/lotto/oop/domain/Rank.java

```diff
@@ -0,0 +1,42 @@
+package lotto.oop.domain;
+
+public enum Rank {
+    FIRST(6, 2000000000),
```

Comment:
```suggestion
    FIRST(6, 2_000_000_000),
```

위와 같이 가독성을 좋게 만들 수 있어요.